cmd/booster-http/run.go

```diff
@@ -76,20 +76,10 @@ var runCmd = &cli.Command{
 			Usage: "enables serving raw pieces",
 			Value: true,
 		},
-		&cli.BoolFlag{
-			Name:  "serve-blocks",
-			Usage: "serve blocks with the ipfs gateway API",
-			Value: true,
-		},
-		&cli.BoolFlag{
-			Name:  "serve-cars",
-			Usage: "serve CAR files with the ipfs gateway API",
-			Value: true,
-		},
-		&cli.BoolFlag{
-			Name:  "serve-files",
-			Usage: "serve original files (eg jpg, mov) with the ipfs gateway API",
-			Value: false,
+		&cli.StringFlag{
+			Name:  "serve-gateway",
+			Usage: "serve deserialized responses with the ipfs gateway API (values: 'verifiable','all','none')",
```

Comment:
I'd suggest we add a Description to StringFlag that describes how the behaviour of booster-http changes when specifying each of the options: verifiable, all and none

Reply:
ack. changes made on the last commit.